Formula/bazel.rb

```diff
@@ -16,11 +16,14 @@ class Bazel < Formula
 
   def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
+    # Force Bazel to put all temporary file in the homebrew cache
+    ENV["BAZEL_WRKDIR"] = "#{HOMEBREW_CACHE}/bazel-workdir"
```

Comment:
Are these files reusable from one build of bazel to the next, or should these files be in the Homebrew temporary directory?